scripts/e2e/config/kserve/overlay.yaml

```diff
@@ -1,24 +1,28 @@
 apiVersion: apps/v1
 kind: StatefulSet
+metadata:
+  name: kserve-controller-manager
+  namespace: kserve
+$patch: delete
+---
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: kserve-controller-manager
   namespace: kserve
 spec:
   template:
     spec:
       containers:
-      - name: manager
-        image: kserve/kserve-controller:v0.9.0
-        resources:
-          limits:
-            cpu: 1
-            memory: 300Mi
-          requests:
-            cpu: 100m
-            memory: 64Mi
-
+        - name: manager
+          resources:
+            limits:
+              cpu: ~
```

Comment:
What is the significance of this symbol, and why is this change made?

Reply:
In YAML, tilde means NULL value [1]. So in Kustomize, it means we remove resources.limits.cpu

https://stackoverflow.com/questions/51990175/what-is-the-purpose-of-tilde-character-in-yaml